src/client/src/containers/NewProject/index.tsx

```diff
@@ -54,7 +54,7 @@ const NewProject = ({
       <div className={styles.body}>
         <FormattedMessage
           id="newProject.body"
-          defaultMessage="Enter a project name and output path, then click next to get started."
+          defaultMessage="Enter a project name and output path, then click next to get started on your full stack application."
```

Comment:
Out of curiosity, why do we need a default message?

Reply:
Hey, its actually used in localization to set the fallback value of the id (newProject.body). So if the value isn't defined for newProject.body in the json, it will gice that message. It automatically populates the en.json with that value after running a command too.